docs/content_management/create_edit_pages.md

```diff
@@ -157,12 +157,23 @@ The following actions are available:
 - Move up block - allows you to change position of the block on the page by moving it up
 - Move down block - allows you to change position of the block on the page by moving it down
 - Block settings - allows you to access properties window
+- Copy - allows you to reuse a configured block across pages by using the Clipboard section.
 - Duplicate - duplicates existing block with its settings
 - Refresh - refreshes preview of the block
 - Delete - deletes existing block
 
 ![Block window - size adjustment](img/block_window_size_adjust.png "Block window - size adjustment")
 
+#### Copy blocks
+
+Copying blocks allows you to reuse configured blocks across different pages.
```

Comment:
```suggestion
You can copy blocks to reuse them on other pages.
```
I'd focus on the reader, not the ssystem.

Reply:
nitpick: you can use it on the same page as well.